platform_design/lib/news_tab.dart

```diff
@@ -18,10 +18,10 @@ class NewsTab extends StatefulWidget {
   const NewsTab({Key? key}) : super(key: key);
 
   @override
-  _NewsTabState createState() => _NewsTabState();
```

Comment:
`State<NewsTab> createState() => _NewsTabState();`

Reply:
Updated all instances in this PR